Blazorade.Bootstrap.Components/ListGroupButton.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Blazorade.Bootstrap.Components
+{
+    public class ListGroupButton : Button
+    {
+        protected override void OnParametersSet()
+        {
+            base.OnParametersSet();
+
+            this.AddClasses(ClassNames.ListGroups.Item, ClassNames.ListGroups.Action);
+
+            // replace the button coloration with the list item coloration
+            this.RemoveClass(this.GetColorClassName(prefix: !this.IsOutline ? ClassNames.Buttons.Button : ClassNames.Buttons.OutlineButton, color: this.Color));
```

Comment:
This is probably the best way of handling it for the moment, but I guess we should at some point figure out a more elegant way to control what classes should not be added to a component in similar cases where the base class adds something that the derived class would like to prevent.

Reply:
I think a revisit of the BootstrapComponentBase and the hierarchy that extends from there might be a good idea. We can define virtual "helper" methods that handle this sort of "if property is set, add the correct class" thing as a base functionality. Then, if we want to do something DIFFERENT in a class further down the hierarchy, we just override that method. That way, in Button, we'd be setting it to one then, but in ListGroupButton, we could override and replace that implementation. That'd be a refactor for another pull request, though.

Reply:
Virtual methods won't help derived classes in picking which classes from the base class it wants to keep, and which it doesn't, unless we create a virtual method for each class that the base class wants to add.

But as I said, your solution is good for now.